pallets/asset/src/lib.rs

```diff
@@ -2577,4 +2590,14 @@ impl<T: Config> Module<T> {
         Self::deposit_event(RawEvent::AssetTypeChanged(did, ticker, asset_type));
         Ok(())
     }
+
+    /// Returns `None` if there's no asset associated to the given ticker,
+    /// returns Some(true) if the asset exists and is of type `AssetType::NFT`, and returns Some(false) otherwise.
+    pub fn nft_asset(ticker: &Ticker) -> Option<bool> {
```

Comment:
Would it better to check if the ticker is in the NFT storage (e.g. `CollectionTicker`) rather than relying on its AssetType?

Reply:
This is used when creating the NFT collection, so that storage wouldn't have been initialized yet.  Might be cleaner if this function just returned the AssetType and the "if asset_type == AssetType::NFT" logic going into the NFT pallet.